pallets/crowdloan-rewards/src/lib.rs

```diff
@@ -10,17 +10,34 @@ mod tests;
 
 #[pallet]
 pub mod pallet {
-	use frame_support::{dispatch::fmt::Debug, pallet_prelude::*, traits::Currency, traits::ExistenceRequirement::AllowDeath};
+	use frame_support::{
+		dispatch::fmt::Debug,
+		pallet_prelude::*,
+		traits::{Currency, ExistenceRequirement::AllowDeath, IsType},
+		PalletId,
+	};
 	use frame_system::pallet_prelude::*;
-	use sp_runtime::traits::{AccountIdConversion, Saturating, Verify};
-	use sp_runtime::{MultiSignature, SaturatedConversion};
 	use sp_core::crypto::AccountId32;
-	use sp_std::{convert::{From, TryInto}, vec::Vec};
+	use sp_runtime::traits::{AccountIdConversion, Saturating, Zero};
+	use sp_runtime::{SaturatedConversion};
+	use sp_std::{
+		convert::{From, TryInto},
+		vec::Vec,
+	};
 
 	#[pallet::config]
-	pub trait Config: frame_system::Config + pallet_treasury::Config {
+	pub trait Config: frame_system::Config {
 		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
 
+		/// The crowdloan's module id, used for deriving its sovereign account ID.
+		type PalletId: Get<PalletId>;
+
+		/// The reward balance.
+		type RewardCurrency: Currency<Self::AccountId>;
+
+		/// Percentage rates of token at Token generating event rate (TGE)
+		const TGE_RATE: u32;
+
 		type RelayChainAccountId:
```

Comment:
don't need anymore

Reply:
Removed it:
https://github.com/polkafoundry/newpolka/pull/55/commits/2ab33c221080d49511dd1ad89c9e64ffa03503e4#diff-2bbae40c9c7fdfeeb0b1708d368844303a595083bc948ccd684b577e5d13f6ebL39